pkg/utils/utils_test.go

```diff
@@ -292,4 +293,15 @@ var _ = Describe("Utils", Label("utils"), func() {
 			Expect(err).To(HaveOccurred())
 		})
 	})
+	Describe("GetUkiCmdline", Label("GetUkiCmdline"), func() {
+		It("returns the default cmdline", func() {
+			cmdline := utils.GetUkiCmdline()
+			Expect(cmdline).To(Equal(constants.UkiCmdline))
+		})
+		It("returns the default cmdline with the cmdline flag", func() {
+			viper.Set("cmdline", "cmdline")
```

Comment:
how about a different value for the key and for the value?

Reply:
done!